src/runner.py

```diff
@@ -0,0 +1,22 @@
+# Модуль, использующий функции из pytasks.py
+def runner(*functions):
+    """Функция импорта из pytasks.py"""
+    if not functions:
+        from pytasks import count_characters
```

Comment:
it's bad practices to import inside function
more of that we can collect all functions in module (maybe we'll add some tasks later)